fix/upp/postxconfig-NT-fv3lam_rrfs.txt

```diff
@@ -7793,7 +7793,7 @@ UEID_ON_EFL
 1
 tmpl4_0
 UEID
-NCEP
+?
```

Comment:
Keep NCEP. Similar for others.

Reply:
Thanks @guoqing-noaa.  I think I may have just found the issue here...I will make another commit...stand by.